scripts/cibuild.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+
+vagrant up --provision
+vagrant ssh -c "
+    cd /vagrant
+    echo -e '\nRunning linter...'
+
+    # run linter with console output
+    npm run lint
+
+    # run linter with file output for Jenkins
+    npm run lint:ci
```

Comment:
Although I don't think that the script is likely to be executed that way in this project, I just mentioned the convention because it's a good security measure for STRTA that may set variables that we don't want leaking into the CI environment.